s/babylon/camera/utils/look-add.ts

```diff
@@ -0,0 +1,8 @@
+import {v2, V2} from "../../../utils/v2.js"
+import {cap} from "../../../utils/numpty.js"
+
+export function lookAdd(state: {currentLook: V2}, vector: V2) {
+	const radian = Math.PI / 2
+	state.currentLook = v2.add(state.currentLook, vector)
+	state.currentLook[1] = cap(state.currentLook[1], -radian, radian)
+}
```

Comment:
let's make this a pure function without side-effect

it should accept a look vector and an addition vector, and return a new result vector -- without writing any side-effect